test/models/tasks/copy_image_task_test.rb

```diff
@@ -8,10 +8,18 @@
   it "has task options" do
     opts = task.task_options
     assert_equal opts[:job_type], "image"
-    assert_equal opts[:source], image.original_url
+    assert_equal opts[:source], image.href
```

Comment:
What does `opts` do? It seems like it's coming from porter.

Reply:
Maybe I'm making that up.

Reply:
I believe this what we're sending _to_ Porter.

Like previously, it was the `image.original_url` (database field).  And Porter would copy that.

But now it's the `image.href` ... which is a method, returning either the `original_url` _OR_ the image's final CDN `http://` location.  Based on what the images status is.

Reply:
I only just now realize that opts was defined here.